### PR TITLE
delete leader rotation signal from banking stage

### PR DESCRIPTION
**Pull request by carllin**
#### Problem
Sending leader rotation signals from both banking stage and tvu causes/leaves potential for races between different events that rely on either one/both of these signals

#### Summary of Changes
Centralize the rotation signals to just the replay_stage

Fixes #

### CODE REVIEW

File: src/fullnode.rs [hunk 1]
```diff
@@ -900,14 +897,8 @@ mod tests {
...
-        info!("Wait for leader -> validator transition");
-        let rotation_signal = leader
-            .rotation_receiver
-            .recv()
-            .expect("signal for leader -> validator transition");
-        debug!("received rotation signal: {:?}", rotation_signal);
-        // Re-send the rotation signal, it'll be received again once the tvu is unpaused
-        leader.rotation_sender.send(rotation_signal).expect("send");
+        // Wait for Tpu bank to progress while the Tvu bank is stuck
+        sleep(Duration::from_millis(1000));
```
Comment (sakridge) [inline src/fullnode.rs]: how is this right? how is 1 second sufficient or too much? Seems kind of random.

Comment (carllin) [reply src/fullnode.rs]: Without the tpu signal, we no longer have a way to check that the tpu is ahead of the tvu, so the wait is just to ensure the tpu can hit the 5 ticks before the tvu starts again. At the default 10 ticks a second, this should be enough time. What might be better is to set the tick rate for the test to be faster?

**Comment by carllin** [comment]
@sakridge, the sleeps and waits are only in two tests that formerly depended on the tpu signal, we can expose some api in the fullnode to check on tpu height instead of using sleeps/waits, but that doesn't seem critical at the moment

**Comment by garious** [comment]
I like this PR because it deletes code, but as @sakridge points out, it adds a long sleep to a test. Adding a sleep() suggests the test will probably be flaky and also means that every developer and every CI run has to pay a price to run a test they may not care much about. Is there a way to detect the state transition such that the sleep() call can be replaced with some sort of deterministic signal that it's time to move forward?  Perhaps the original code is exactly that?

**Comment by sakridge** [comment]
@carllin ok, I guess I didn't get that the signals are there just for the tests. This seems fine, although would prefer not just waiting for 1 second.

**Pull request merged**